translations/cs/LC_MESSAGES/wesnoth-ala.po

```diff
@@ -1705,7 +1709,7 @@ msgid ""
 "friends ogres... I believe... I don't remember well!"
 msgstr ""
 "Glubtrub nemít rád hady! Málo tuku k jídlu, a ne moc dobré! Chtít zabít naše "
-"malé, když si hrát poblíž ruin! Ale voják Wesnoth barvy nikdy nebýt přítel "
+"malé, když si hrát poblíž ruin! Ale voják Wetspot barvy nikdy nebýt přítel "
```

Comment:
unify with the same use above
```suggestion
"malé, když si hrát poblíž ruin! Ale voják wetspot barvy nikdy nebýt přítel "
```

Reply:
I'm trying to ensure "Wetspot" is consistently capitalized in English; if it isn't in the original English string, then that should be fixed.

Reply:
As cooljeanius says. We may reconsider it later, but I would leave it for now.

Reply:
But @MultiSeptim uses Wetspot in Czech translation as an adjective, so it should be with lower case (in Czech). Anyway, the adjective is crippled (non-inflective, without case) and does not reflect original funny meaning. So, what about Vesnut (ve snu ~ in a dream), vesnutí barvy, barvy Vesnutu (colors of Wetspot)...?

Reply:
Vesnut jó!